### ISSUE TITLE
Problems about the test results and the number of categories for Cityscapes

### ISSUE THREAD

**Issue by zhuqingweiyu** [fyu/drn#10]
Hi, @fyu ,

Thanks for your code.
When I downloaded the Cityscapes dataset, I found the number of classes is 30， not 19 in the README.md. So I can't reproduce the mAP of the validation set. I am so confused.

And I am also confused about the difference between IOUclass and IOU category in the evaluation for Cityscapes. 

Thanks.

**Comment by sabSAThai** [comment]
Hi @fyu, 
The number of labels in Cityscapes dataset is 34 (35 if you include the -1 label of the license plate).
Attached pic is the output of the helper scripts given by Cityscape creators. [Link to the script](https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py)

![screenshot from 2017-10-26 04-52-03](https://user-images.githubusercontent.com/16862760/32028067-1cff90e2-b9dc-11e7-8d5c-1691be56cb29.png)

Due to this mismatch of the number of labels, the code in which I am running a validation phase spits out the following error when it calculates the loss.
![screenshot from 2017-10-26 05-01-06](https://user-images.githubusercontent.com/16862760/32028188-b731a43e-b9dc-11e7-92e9-db3da275b2c6.png)
 
Please look into this.

**Comment by fyu** [comment]
The model is trained with 19 labels, i.e. categories with valid trainId,  following the official segmentation evaluation protocal of cityscapes.
